apps/web/src/modules/job/post/components/ShootingForm/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface ShootingFormProps {
+  id: string
+  index: number
+  remove: (idx: number) => void
+  //pls forgive me for this
+  control: any
```

Comment:
I think we can specify this type (but not in easy way)

Reply:
`Control<IPostJobSchemaType>`